pkg/types/validation/installconfig_test.go

```diff
@@ -235,7 +234,7 @@ func TestValidateInstallConfig(t *testing.T) {
 				}
 				return c
 			}(),
-			valid: true,
+			expectedError: `^platform: Invalid value: types\.Platform{AWS:\(\*aws\.Platform\)\(nil\), Libvirt:\(\*libvirt\.Platform\)\(0x[0-9a-f]*\), OpenStack:\(\*openstack\.Platform\)\(nil\)}: must specify one of the platforms \(aws, openstack\)$`,
```

Comment:
This test originally verified that a valid libvirt platform passed validation. Now, the test verifies that libvirt is not a valid platform. If this is the test that we want, then we should change the name of this test and remove the subsequent equivalent test. I would argue that we still want a test that verifies what this test originally verified, but, given that the libvirt platform is only for development purpose, I'm willing to be convinced otherwise.

Reply:
> Now, the test verifies that libvirt is not a valid platform.

Now that libvirt validity depends on build tags, I think we want the regexps to distinguish between "libvirt not compiled in as a valid platform, but if it had been, this would have been fine" (this test) and "libvirt not compiled in as a valid platform, and even if it had been, this would still have been broken" (the later tests).  Otherwise the deeper libvirt validation isn't covered by tests.

Reply:
That sounds like a reasonably suggestion for what the tests could do.